receiver/filereceiver/README.md

```diff
@@ -0,0 +1,28 @@
+# File Receiver
+
+| Status                   |               |
+|--------------------------|---------------|
+| Stability                | [development] |
+| Supported pipeline types | metrics       |
+| Distributions            | [contrib]     |
+
+The File Receiver reads the output of a
+[File Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter),
+converting that output to metrics, and sending the metrics down the pipeline.
+
+Currently, the only file format supported is the File Exporter's JSON format.
+Reading compressed output, rotated files, or telemetry other than metrics are not supported at this time.
+
+## Getting Started
+
+The following settings are required:
+
+- `path` [no default]: the file containing the metrics written by a File Exporter.
```

Comment:
What should happen if the path doesn't exist or the file doesn't exist? Is it expected for the path to exist?

Reply:
If the file doesn't exist there will be nothing for this receiver to do, since that file is the only input source for this receiver. I'm thinking that if the file doesn't exist, the receiver should both fail to start and produce an informative error.

Reply:
Since the exporter supports rotation, I suggest that the receiver should also. Would it be possible to use the `pkg/stanza/fileconsumer` package?

Reply:
I don't mind the course of action provided the behaviour is clear to the user :)